docs/photoniq/est/manage-est-streams.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 40
+title: Monitor and Debug P3 with EST Streams
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Stream feature in Edge Side Tagging (EST) is designed to enhance your tracking and debugging capabilities for third-party tags. This guide explains how to use this feature for debugging and analytics.
+
+:::note
+Keep the Stream feature enabled for debugging purposes only when necessary to minimize performance impact.
+:::
+
+## Understanding the EST Stream Feature
+
+Turning on EST Stream feature provides access to stream data in real time and the Debug tab in the EST dashboard.
```

Comment:
EST Stream feature will not provide access to stream data , instead its for debug purpose only